frontend/src/app/auth.guard.ts

```diff
@@ -0,0 +1,19 @@
+import { JwtHelperService } from '@auth0/angular-jwt';
+import { Injectable } from '@angular/core';
+import { CanActivate, Router } from '@angular/router';
+
+@Injectable()
+export class AuthGuard implements CanActivate {
+  constructor( private router: Router) {
+  }
+  private jwtHelper = new JwtHelperService();
+  canActivate() {
+    let token = localStorage.getItem('jwt');
```

Comment:
to czy użytkownik jest zalogowany wydziel do serwisu, a w guardzie sprawdź tylko co serwis zwróci